numpy/_core/tests/test_regression.py

```diff
@@ -453,7 +453,7 @@ def test_lexsort_zerolen_element(self):
 
     def test_pickle_py2_bytes_encoding(self):
         # Check that arrays and scalars pickled on Py2 are
-        # unpickleable on Py3 using encoding='bytes'
+        # unpicklable on Py3 using encoding='bytes'
```

Comment:
Leave this one alone. Either form is acceptable.

Reply:
it was changed in pandas

but again if you still think it should be left alone it can easily be reverted

Reply:
https://en.wiktionary.org/wiki/pickleable

Yes, please leave it alone.